### PR TITLE
Improve page aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the new behavior (if this is a feature change)?**
- The way that data is being queried to the page has been changed greatly.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
LGTM

File: src/client/pages/_app.js [hunk 1]
```diff
@@ -129,6 +137,7 @@ class MyApp extends App {
 //#region > Exports
 export default MyApp;
 //#endregion
+//#endregion
```
Comment (kleberbaum) [inline src/client/pages/_app.js]:
```suggestion
//#endregion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**